spring-boot-project/spring-boot-devtools/src/main/java/org/springframework/boot/devtools/env/DevToolsHomePropertiesPostProcessor.java

```diff
@@ -61,6 +61,17 @@ public void postProcessEnvironment(ConfigurableEnvironment environment, SpringAp
 		}
 	}
 
+	protected File getProjectRootFolder() {
```

Comment:
It seems a breaking change.
When `PropertyFile` of `HomeFolder` have a property which not exist in `ProjectRootFolder`, after this change the property will be invalid.

Reply:
I don't think this is a breaking change as you have to opt in to looking in the new location by setting that system property or environment variable.